frontend/base/src/main/java/com/virtuslab/gitmachete/frontend/vfsutils/GitVfsUtils.java

```diff
@@ -1,5 +1,6 @@
 package com.virtuslab.gitmachete.frontend.vfsutils;
```

Comment:
do we still need to depend on vavr in all modules? it cannot be dropped anywhere yet? or can it be?

Reply:
Checked all the modules I touched here, every one of them still uses at least one class from vavr. So yes, we need the dependencies.

Reply:
ok, thanks!